app.py

```diff
@@ -6,15 +6,20 @@
 import redis
 
 from flask import Flask, jsonify, send_from_directory
+from flask_cors import CORS
 app = Flask(__name__, static_url_path='', static_folder='static')
+CORS(app)
```

Comment:
CORS should be unnecessary if you're serving the frontend from the same origin as the server.